src/ServiceContainer/SymfonyExtension.php

```diff
@@ -176,12 +178,25 @@ private function loadMinkParameters(ContainerBuilder $container): void
     private function loadBootstrap(?string $bootstrap): void
     {
         if ($bootstrap === null) {
+            $this->loadEnv();
```

Comment:
Maybe instead of loading it in loadBootstrap method, to load it separately just before calling loadBoostrap, to make a check if bootstrap is not specified to try to load environment variables.

Just to have a separated context. What do you think?

Reply:
yes I think you are right, since the bootstrap.php file has disappeared

Reply:
> What you mean has disappeared

The file was config/bootstrap.php

yes of course nothing should be changed to keep a retro compatibility

I agree with you